tests/Integrations/Laravel/V4/EloquentTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\Laravel\V4;
+
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Common\SpanAssertionTrait;
+use DDTrace\Tests\Common\TracerTestTrait;
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+class EloquentTest extends WebFrameworkTestCase
+{
+    use TracerTestTrait, SpanAssertionTrait;
+
+    const IS_SANDBOX = false;
+
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../../Frameworks/Laravel/Version_4_2/public/index.php';
+    }
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->connection()->exec("DELETE from users where email LIKE 'test-user-%'");
+    }
+
+    protected function expectedServiceName()
+    {
+        // Shouldn't this not be set by teh span encoder?
```

Comment:
The SpanEncoder is only used for userland spans. C-level spans run through [the serializer](https://github.com/DataDog/dd-trace-php/blob/8ca38744500482fe9c2d6e642f227012f49e5fd1/src/ext/serializer.c#L314-L342). Once we get the service name worked out (from the earlier comment) the service names should be the same for sandboxed and non-sandboxed integrations. :)